src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -163,6 +167,24 @@ public function text($message)
     /**
      * {@inheritdoc}
      */
+    public function comment($message)
+    {
+        $this->autoPrependText();
+
+        if (!is_array($message)) {
```

Comment:
I like this proposal. If @kbond (the original author of this code) doesn't say anything against it, I'll do these changes. Thanks!